nifi-nar-bundles/nifi-scripting-bundle/nifi-scripting-processors/pom.xml

```diff
@@ -120,28 +120,34 @@
             <scope>test</scope>
         </dependency>
         <dependency>
-            <groupId>org.apache.groovy</groupId>
+            <groupId>org.xmlunit</groupId>
+            <artifactId>xmlunit-core</artifactId>
```

Comment:
Is this dependency required?

Reply:
Yes I use it in place of the Groovy script which did assertions after using XMLSlurper

Reply:
Thanks for clarifying. This sounds like a case where the tests should be simplified. There is nothing specific to XML parsing in the Execute Script components themselves, so there is no need to retain the complex XML parsing tests. If this requires removing some test methods, it would be better to do that and remove this dependency.

Reply:
I believe the output is XML hence there was XML parsing used originally to access the XML. 
The specific place I use it is in `ScriptedRecordSetWriterTest.testRecordWriterGroovyScript`

Reply:
Thanks for the pointer. In that case, it would be better to change the assert to just parse the XML Document using the Java DocumentBuilder, and skip the XPath assertions. That should be sufficient in this particular scenario, and it avoids the additional dependency for now.

Reply:
@exceptionfactory I think I have a way of using Xpath and not using the dependency.

Reply:
@exceptionfactory I removed the xmlunit as you requested but I was able to get the XPath working with plain Java using their Xpath API. IS that okay?

Reply:
Yes! Thanks for making the adjustment, that looks good!